### PR TITLE
fix: homepage redirects to index.html

### LINKED ISSUE #170: Minor Bug history3.html
**Describe the bug**
The home button on the bottom of the page directs to http://www.sack.lk
**To Reproduce**

**Expected behavior**
It should go back to index.html

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [Windows 10 Home]
 - Browser [Chrome]

**Smartphone (please complete the following information):**

**Additional context**

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by sachincool**
## Purpose

The purpose of this PR is to fix #170 

## Goals
homepage Redirects to `index.html` instead of `http://www.sack.lk`
## Approach
I add the change in header and footer as well

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: assets/content/footer.html [hunk 1]
```diff
@@ -13,19 +13,19 @@
...
                     <li><i class="fa fa-angle-right"></i><a href="https://www.sack.edu.lk/about_us.html">About Us</a>
                     </li>
                     <li><i class="fa fa-angle-right"></i><a href="https://www.sack.edu.lk/history1.html">History</a>
                     </li>
                     <li><i class="fa fa-angle-right"></i><a href="https://www.sack.edu.lk/contact_us.html">Contact
-                        Us</a></li>
+                            Us</a></li>
                     <!--<li><i class="fa fa-angle-right"></i><a href="http://www.sack.edu.lk/News">News</a></li>-->
                     <li><i class="fa fa-angle-right"></i><a
                             href="https://www.sack.edu.lk/apps/assignments/">Assignments</a>
```
Comment (Gimhan-minion) [inline assets/content/footer.html:31]: @sathsarabandaraj Can you make these links to refer in the directory itself too? Just like you did link the index file.

File: assets/content/header.html [hunk 2]
```diff
@@ -39,13 +39,13 @@
                     <span class="icon-bar"></span>
                 </button>
                 <a class="navbar-brand" href="http://www.sack.edu.lk"><img src="https://www.sack.edu.lk/img/logo.png"
-                                                                           alt="St. Anthony's College"></a>
+                        alt="St. Anthony's College"></a>
```
Comment (Gimhan-minion) [inline assets/content/header.html:42]: You can do the above-mentioned changes here too 🤗

**Review by Gimhan-minion — CHANGES_REQUESTED** [review]
Good job @sathsarabandaraj Small changes are requested

**Comment by sachincool** [comment]
@Gimhan-minion I found that using `index.html` for relative path breaks for certain URLs like `apps/assignments` 
where it will resolve to `apps/assignments/index.html` this is not expected outcome. so, I shifted the paths to `../../` which works even for nested roots or just normal homepage due to how express resolves these paths. It's a hack but it works.

**Comment by Gimhan-minion** [comment]
> @Gimhan-minion I found that using `index.html` for relative path breaks for certain URLs like `apps/assignments`
> where it will resolve to `apps/assignments/index.html` this is not expected outcome. so, I shifted the paths to `../../` which works even for nested roots or just normal homepage due to how express resolves these paths. It's a hack but it works.

Actually this was the change I requested! Anyhow, you read my mind! good job @sachincool Thank you! :hugs:

**Review by Gimhan-minion — APPROVED** [review]

**Pull request merged**